cider/exec/operator/join/CiderStdUnorderedHashTable.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+// used as test
+template <typename Key, typename T>
+class Dup_map {
+ public:
+  Dup_map(){};
+  std::vector<T> lookup(Key key) {
+    auto iter = umap_.find(key);
+    if (iter != umap_.end()) {
+      return iter->second;
+    }
+    std::vector<T> empty_res;
+    return empty_res;
+  }
+
+  void insert(Key&& key, T&& value) {
```

Comment:
always insert right reference?

Reply:
This is just for test and may used for benchmark. I thought right reference is fast. Correct me if wrong.